src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,30 @@
+.forecast-summary{
+  padding: 5%;
+  background: #66748A;
+  border-radius: 25px;
+  
+  color:white;
+}
+
+.forecast-summary__button{
+  box-shadow: 0px 0px 0px transparent;
+  border: 0px solid transparent;
+  background:transparent;
+  text-shadow: 0px 0px 0px transparent;
+}
+
+.forecast-summary:hover{
+  background: #404756;
+  color:white;
+  cursor: pointer;
+}
+
+.forecast-summary:hover > .forecast-summary__button{
+  color:white;
+  cursor: pointer;
```

Comment:
This line can be deleted as `forecast-summary:hover` already adds the cursor: pointer declaration